doc/sphinx/user-extensions/syntax-extensions.rst

```diff
@@ -527,7 +527,7 @@ is just an identifier, one could have said
 ``p at level 99 as strict pattern``.
 
 Note also that in the absence of a ``as ident``, ``as strict pattern`` or
-``as pattern`` modifiers, the default is to consider sub-expressions occurring
+``as pattern`` :token:`modifier`\s, the default is to consider sub-expressions occurring
```

Comment:
Are you intending to escape the `s` here

Reply:
Same answer as above.